kernel/exit.c

```diff
@@ -25,3 +25,9 @@ void solo5_exit(int status)
     log(INFO, "Solo5: solo5_exit(%d) called\n", status);
     platform_exit(status);
 }
+
+void solo5_abort(void)
```

Comment:
This can just call directly through to `platform_exit(SOLO5_EXIT_ABORT, NULL)` (See summary).